From 5d241f2f7722307a2bd2b93db260323cb9cdfb7b Mon Sep 17 00:00:00 2001
From: Samuel Ortiz <sameo@linux.intel.com>
Date: Fri, 25 Jul 2014 15:18:02 +0200
Subject: [PATCH 2/2] misc: dgnss: Add BCM4752E support

Some devices export BCM4752 GPS with external LNA as BCM4752E through
the ACPI DSDT.

Change-Id: I1200520279bdb12f54ad762e21f0a8a8641b8bde
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
---
 drivers/misc/dgnss/dgnss.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/misc/dgnss/dgnss.c b/drivers/misc/dgnss/dgnss.c
index 2db0084..9564fa5 100644
--- a/drivers/misc/dgnss/dgnss.c
+++ b/drivers/misc/dgnss/dgnss.c
@@ -26,6 +26,7 @@
 
 #ifdef CONFIG_ACPI
 #define ACPI_DEVICE_ID_BCM4752 "BCM4752"
+#define ACPI_DEVICE_ID_BCM4752E "BCM4752E"
 #endif
 
 static int dgnss_enable_status;
@@ -155,6 +156,7 @@ static void dgnss_shutdown(struct platform_device *pdev)
 static const struct acpi_device_id acpi_gps_id_table[] = {
 	/* ACPI IDs here */
 	{ACPI_DEVICE_ID_BCM4752},
+	{ACPI_DEVICE_ID_BCM4752E},
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, acpi_gps_id_table);
-- 
1.8.3.2

